packages/editor/src/client/route.tsx

```diff
@@ -88,6 +92,16 @@ export const AuthorEditRoute = route(
 )
 export const AuthorCreateRoute = route(RouteType.AuthorCreate, routePath`/author/create`)
 
+export const NavigationCreateRoute = route(
+  RouteType.NavigationCreate,
+  routePath`/navigation/create`
+)
+export const NavigationListRoute = route(RouteType.NavigationList, routePath`/navigation`)
```

Comment:
The path should be `/navigations` check with the other routes.

Reply:
ok, done